keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -731,6 +732,12 @@ feature flags will be enabled on some API servers and not others
 during the rollout. Similarly, consider large clusters and how
 enablement/disablement will rollout across nodes. -->
 
+During a rollout, some apiservers may support aggregated discovery and
+some may not. It is recommended that clients request for both the
+aggregated discovery document with a fallback to the unaggregated
+discovery format. This can be achieved by setting the Accept header to
+have a fallback to the default GVK of the `/apis` and `/api` endpoint.
```

Comment:
And this is already implemented in client-go? The server already handles aggregated apiservers that don't do aggregated?

Reply:
I'll add an additional note for aggregated apiservers. Yes this is implemented in client-go but there may be non go clients who are interested in supporting aggregated discovery and part of this is aimed at those clients.